src/libsystemd/sd-json/sd-json.c

```diff
@@ -1477,7 +1477,8 @@ _public_ int sd_json_variant_equal(sd_json_variant *a, sd_json_variant *b) {
 }
 
 _public_ void sd_json_variant_sensitive(sd_json_variant *v) {
-        assert(v);
+        if (!v)
+                return;
```

Comment:
I think we should use assert_return here, since this is a setter.

Reply:
for public apis we are usually relatively friendly. And conceptually, turning on sensitivity of a NULL object is kinda redundant. If you have no data, no data can be linked. Hence I figured we should be nice to users here and just make it a NOP.